### DISCUSSION TITLE
Parse JSON

### DISCUSSION THREAD

**Discussion by EduardoZeroum** [Budibase/budibase#6483] [Help] [answered]
Hi, i need help with my budibase project, the problem is i managed to get the string associated from a client code , but when i'm returning the string, it shows up something like this "{"data":[{"nome":"Consumidor Final"}]}" and i want to return "Consumidor Final" i tried parseJSON with javascript but he returns "[object Object]" and i dont know if i have to format it in the budibase or in the query, can someone help me? I will disponibilize the prints of my project, thank you.
![image](https://user-images.githubusercontent.com/104850197/176153869-599917f2-8c3b-4ede-9b5b-4fe95e78e668.png)
![image](https://user-images.githubusercontent.com/104850197/176153954-73ebac11-3843-460a-b2c7-d526f85d80ec.png)
![image](https://user-images.githubusercontent.com/104850197/176154018-9e3d7160-e7de-4af7-8869-314b1aeb7ec4.png)

**Comment by melohagan** [comment, marked as answer]
Hey @EduardoZeroum 

No need to parse JSON, you can use handlebars.
As per this docs page (https://docs.budibase.com/docs/referencing) you need to use `.data.0.nome` to access the first rows name

**Reply by EduardoZeroum** [reply]
Thank you so much, it worked!

**Reply by matteoveglia** [reply]
For anyone coming back to this post now, for some reason the referencing page has been removed from the documentation, here's a handy wayback machine link
https://web.archive.org/web/20220601090458/https://docs.budibase.com/docs/referencing

**Reply by melohagan** [reply]
Thanks @matteoveglia The section was moved here: https://docs.budibase.com/docs/chaining-actions#action-referencing
